repository/c.json

```diff
@@ -3112,6 +3112,18 @@
 				}
 			]
 		},
+		{
+			"name": "Comments-only Color Scheme",
+			"description" : "Colorscheme without syntax highlighting",
```

Comment:
You don't need the description here, it will pick up the one on the repo. Makes it easier to change too.

Reply:
```suggestion
```

Reply:
Fixed.